### DISCUSSION TITLE
AE90A022-4B79-1DB0-CDA4-F38EF4DB8F58

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#72] [General]
# AE90A022-4B79-1DB0-CDA4-F38EF4DB8F58

The mysterious monolith designated as TMA-2 (incorrectly named after TMA-1, the monolith discovered on the Moon) is found orbiting Jupiter near Io. This is the main destination of both the failed Discovery One mission in 2001 and the joint US-Soviet Alexei Leonov mission in 2010. The monolith mysteriously disappeared 2-15 days before Jupiter became a star known as Lucifer

In the 2010: Odyssey Two (novel), TMA-2 was also nicknamed as 

https://celestia.mobi/addon?item=AE90A022-4B79-1DB0-CDA4-F38EF4DB8F58

**Comment by galileokeplertycho** [comment]
plesae add DISCOVERY ship !!!
